Tests/LibCore/TestLibCoreStream.cpp

```diff
@@ -459,13 +389,11 @@ TEST_CASE(buffered_small_file_read)
     auto buffer = ByteBuffer::create_uninitialized(4096).release_value();
     for (auto const& line : expected_lines) {
         VERIFY(file->can_read_line().release_value());
-        auto maybe_read_line = file->read_line(buffer);
-        EXPECT(!maybe_read_line.is_error());
-        EXPECT_EQ(maybe_read_line.value().length(), line.length());
-        EXPECT_EQ(StringView(buffer.span().trim(maybe_read_line.value().length())), line);
+        auto read_line = TRY_OR_FAIL(file->read_line(buffer));
+        EXPECT_EQ(read_line.length(), line.length());
+        EXPECT_EQ(StringView(buffer.span().trim(read_line.length())), line);
     }
-    EXPECT(!file->can_read_line().is_error());
-    EXPECT(!file->can_read_line().value());
+    EXPECT(!TRY_OR_FAIL(file->can_read_line()));
```

Comment:
This would read clearer to me as 2 lines, like you did below in the buffered_file_tell_and_seek test.

```
bool can_read_line = TRY_OR_FAIL(file->can_read_line());
EXPECT(!can_read_line);
```

Reply:
Absolutely, thanks!